draft-ietf-masque-connect-udp-listen.md

```diff
@@ -153,11 +170,103 @@ UDP Payload:
 : The unmodified UDP Payload of this proxied UDP packet (referred to as "data
 octets" in {{UDP}}).
 
+# Context ID {#contextid}
+
+Context ID values MUST always be greater than 0. A payload with value 0 MUST
+be dropped by the receiving party.
+
+An even context ID MUST be used by clients to send packets to the proxy, and an odd
+context ID MUST be used by the proxy to send packets to the client. A Client and
+a proxy MUST exhange COMPRESSION_ASSIGN packets for a given context ID to guarantee
+that all payloads are received and processed by the other party, see
+{{compression}}. They MAY pre-emptively use Context IDs not acknowledged by the
+other party yet, knowing that those packets MAY be lost since the client is not
+guaranteed to be ready to receive until it receives the acknowledgement
+COMPRESSION_ASSIGN.
+
+## Address Compression {#compression}
+
+If the client wishes to allocate a Context ID for uncompressed packets,
```

Comment:
Now that we've changed how the uncompressed context ID is sent, can you do a search for all instances of the header? I saw one earlier that still says that's how we send the context and that's incorrect since now we just use the header to indicate support for this extension